etcd/map.jinja

```diff
@@ -1,6 +1,6 @@
 {% set server = salt['grains.filter_by']({
     'Debian': {
-        'pkgs': ['etcd', 'python-etcd'],
```

Comment:
Etcd packages are being built and are no longer supported. Should be removed like in salt-formula-kubernetes.